### DISCUSSION TITLE
Error After Build: Drizzle ORM and Better SQLite 3

### DISCUSSION THREAD

**Discussion by VitorAndrey** [alex8088/electron-vite#406] [Q&A]
Hello Electron Vite community,

I'm currently facing an issue with my Electron Vite application, and I'm hoping someone can provide some guidance. I'm using Better SQLite 3 with Drizzle ORM, and I encounter an error only after building the application.

### Problem Overview:
- **Scenario:** My application works perfectly fine during development (after running `npm run dev`), but I encounter an error after building the application for Windows (using `npm run build:win`).
- **Error Message:** The specific error message mentions the inability to find the "meta/_journal.json" file.

### Additional Context:
- **Database Setup:** I'm utilizing Better SQLite 3 in combination with Drizzle ORM for database operations.
- **Build Goal:** I aim for my app to work seamlessly without requiring users to install additional dependencies.

### Screenshots:
- Working in development: <br />
![Working](https://github.com/alex8088/electron-vite/assets/114686823/4efa1027-0b8b-4f3c-9f2d-f6e1f8fa3983)
- Error after build: <br />
![Error](https://github.com/alex8088/electron-vite/assets/114686823/7e49ad55-5e1a-467c-9f54-c9b8e0dbceda)

I'm seeking tips or suggestions on where to start investigating this issue. Could it be related to Drizzle ORM, Better SQLite 3, or something else in the build process? Any insights or experiences with similar issues would be greatly appreciated.

Thank you for your time and assistance!

**Comment by alex8088** [comment]
You should install the related dependence in deps

https://electron-vite.org/guide/dev#dependencies-vs-devdependencies
